### PR TITLE
Wp 681 refactor consul templating

### PR DESCRIPTION
**Pull request by ahogui12**
Update and replace https://github.com/wazo-platform/ansible-role-consul-service/pull/2
From a list of URLs: [url1, url2,...] , adds {.., URL_auto_1: xxx.test.com, URL_auto_2: xxz.test.com} in Tags in Consul

### CODE REVIEW

File: templates/consul_service.json.j2 [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "service": {
         "name": "{{ consul_service_name }}",
         "id": "{{ consul_service_name }}-{{ metadata_id }}",
-        "tags": [{{ consul_service_tags | map("to_json") | join(', ') }}],
+        "tags": [{{ consul_service_tags | join(', ') }}],
```
Comment (sduthil) [inline templates/consul_service.json.j2:5]: Not sure to understand why this change?

Comment (ahogui12) [reply templates/consul_service.json.j2]: The change was due for tests, sorry for pushing it

**Review by sduthil — APPROVED** [review]

**Review by sduthil — APPROVED** [review]
